composer/datasets/cifar.py

```diff
@@ -49,10 +68,84 @@ def initialize_object(self, batch_size: int, dataloader_hparams: DataLoaderHpara
                 memory_format=self.synthetic_memory_format,
             )
 
+        elif self.use_ffcv:
+            try:
+                import ffcv  # type: ignore
```

Comment:
Curious why the type ignore?

Reply:
Was getting pyright error because these were conditional imports.

Reply:
Since ffcv is not installed by default, pyright throws lint error without ignore.